docs/prepping-local-system-runbook-linux.md

```diff
@@ -228,176 +217,167 @@ This gcloud configuration is called [default]. You can create additional configu
 
-4. Verify that you now have the key with the fingerprint `9DC8 5822 9FC7 DD38
-    854A E2D8 8D81 803C 0EBF CD88` by searching for the last 8 characters of the
-    fingerprint.
+### For EKS only: installing AWS CLI version 2
 
-    `$ sudo apt-key fingerprint 0EBFCD88`
+Install the AWS CLI utility with a packaging tool such as apt-get.
 
-    The command should return:
+`$ sudo apt-get install awscli`
+
+To confirm the installation or upgrade, enter:
 
 ```bash
-pub 4096R/0EBFCD88 2017-02-22
-9DC8 5822 9FC7 DD38 854A E2D8 8D81 803C 0EBF CD88
-uid Docker Release (CE deb) <docker@docker.com>
-sub 4096R/F273FCD8 2017-02-22
+$ aws --version
+aws-cli/2.0.10 Python/3.7.3 Linux/5.3.0-51-generic botocore/2.0.0dev14
 ```
 
-5. Use the following command to set up the stable repository. You always need the stable repository, even if you want to install builds from the edge or test repositories as well. To add the edge or test repository, add the word "edge" or "test" (or both) after the word stable in the commands below.
-
-`$ sudo add-apt-repository "deb [arch=amd64]
-https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"`
+### For EKS only: installing AWS IAM authenticator
 
-Note: Starting with Docker 17.06, stable releases are also pushed to the edge
-and test repositories.
+Pega recommends using the AWS IAM Authenticator for Kubernetes to authenticate with your Kubernetes cluster using your AWS credentials. You must download the binary file from AWS and then install the CLI on your local computer.
 
-**INSTALL DOCKER CE**
-
-1. Update the apt package index.
+1. To download the latest binary from the AWS site, from your home directory, enter:
 
-    `$ sudo apt-get update`
+```bash
+   $ curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.15.10/2020-02-22/bin/linux/amd64/aws-iam-authenticator
+   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                  Dload  Upload   Total   Spent    Left  Speed
+   100 33.6M  100 33.6M    0     0  1996k      0  0:00:17  0:00:17 --:--:-- 4786k
+```
 
-2. Install the latest version of Docker CE or go to the next step to install a
-    specific version. Any existing installation of Docker is replaced.
+2. To make the AWS IAM Authenticator binary executable, enter:
 
-    `$ sudo apt-get install docker-ce docker-ce-cli containerd.io`
+    `$ chmod +x ./aws-iam-authenticator`
 
-3. Verify that Docker Engine - Community is installed correctly by running the
-    hello-world image.
+3. To use the AWS recommended command to copy the AWS IAM Authenticator file to a newly created folder, `$HOME/bin/aws-iam-authenticator`, and ensure that $HOME/bin comes first in your $PATH, enter:
 
-    `$ sudo docker run hello-world`
+   `$ mkdir -p $HOME/bin && cp ./aws-iam-authenticator $HOME/bin/aws-iam-authenticator && export PATH=$PATH:$HOME/bin`
```

Comment:
This command will not actually put the newly created directory at the beginning of the path.  I'll take a look at the link to see if there is a better way to state these instructions.

Reply:
I took the wording from AWS and refer to the AWS documentation. I think the doc as written for now is ok.